### ISSUE TITLE
Request to have this package produce a migration file, rather than making the changes

### ISSUE THREAD

**Issue by SlyDave** [AXN-Informatique/laravel-pk-int-to-bigint#3]
it would be nice if this package produced an actual migration of the result, that can then be committed, rather than having a migration that runs the artisan command.

By including a migration that runs the artisan commend, there is a requirement to always include this package as a dependency, even after it has served its purpose. 

It would be more useful, if this package produced a migration that can then be committed to the project, that contained all the up's and down's to go from int to bigint and back

**Comment by forxer** [comment]
It's a good idea, it always bothered me to have to install the package when it's only used once.

We will think about it.

**Comment by godismyjudge95** [comment]
For those wanting a standalone migration of this package:
https://gist.github.com/godismyjudge95/12c3cebeb3156c84fbac795a8f803382

@forxer feel free to copy it back into the repo, just don't have the time atm for a PR

**Comment by forxer** [comment]
@godismyjudge95 I don't think this really answers the initial request which was, if I understood correctly, that the package could generate a migration with the up() and down() methods so that there is actually the generated SQL code.
